packages/ui/src/ui/pages/navigation/modals/AttributesEditor.tsx

```diff
@@ -261,6 +273,7 @@ function AttributesEditorLoaded() {
                         hasStaticTables && storage.runMerge,
                     ),
                 );
+                dispatch(ytApi.util.invalidateTags([String(YTApiId.navigationGetAnnotation)]));
```

Comment:
Is it possible to read providedTags from `descriptionApi`?
If not maybe it is better to export `invalidateDescriptionApi` action near `descriptionApi`